core/model/modx/processors/resource/trash/getlist.class.php

```diff
@@ -59,15 +59,18 @@ public function prepareQueryBeforeCount(xPDOQuery $c)
         if (!empty($context)) {
             $c->where(array('modResource.context_key' => $context));
         }
-        $c->where(array('modResource.id:IN' => $this->getDeleted()));
+        if ($deleted = $this->getDeleted()) {
+            $c->where(array('modResource.id:IN' => $deleted));
+        } else {
+            $c->where(array('modResource.deleted' => true));
```

Comment:
This fix should get an empty result a different way. I would suggest:

```
$c->where(array('modResource.id' => 0));
```

The query in getDeleted could get changed later on to something, that does not only check the deleted state.

Reply:
Done.

Reply:
Thanks!